### ISSUE TITLE
Failed github actions check, but not locally 

### ISSUE THREAD

**Issue by pdmthorsrud** [fredrikhgrelland/terraform-nomad-redash#6]
https://github.com/fredrikhgrelland/terraform-nomad-redash/pull/3/checks?check_run_id=1037253070

Timed out on github actions. Email, redis, and postgres all come up fine, but then it times out on redash server. Did a run on local machine with the same parameters, it all worked fine. Just restarted the github actions to see if it's reproducable. Will update

**Comment by pdmthorsrud** [comment]
Well hey, it's working now. I do not know what was wrong, but the redash module seems to be a little unstable when run by github actions (never failed locally yet). Let's just make sure we keep an eye on it. @Neha-Sinha2305 @claesgill 
https://github.com/fredrikhgrelland/terraform-nomad-redash/runs/1038115705?check_suite_focus=true

Closing this for now.

**Comment by pdmthorsrud** [comment]
Nope, never mind. Still not working. Investigating locally right now.
https://github.com/fredrikhgrelland/terraform-nomad-redash/runs/1038115705?check_suite_focus=true

**Comment by pdmthorsrud** [comment]
Update: everything seems to be working locally. Calling it a day, getting back to it tomorrow!
![image](https://user-images.githubusercontent.com/51820995/91487911-ca0ca000-e8ae-11ea-980b-0e18bfb61c76.png)
![image](https://user-images.githubusercontent.com/51820995/91487940-d4c73500-e8ae-11ea-8e0c-f49cce49fcd8.png)

**Comment by pdmthorsrud** [comment]
When reading the logs it seems email redis and postgres take twice as long to create compared to when the run is successful. There does not seem to be a correlation between the parameters of the matrix and what run fails (I ran it three times and they were all different)

**Comment by pdmthorsrud** [comment]
If it turns out this is still working on our local computers we might have to just ignore it for now so that we can get it all done today
@Neha-Sinha2305 @claesgill

**Comment by fredrikhgrelland** [comment]
This might simply be a resource contention Issue. We are not staggering execution of these six docker tasks

**Comment by pdmthorsrud** [comment]
https://github.com/fredrikhgrelland/vagrant-hashistack/issues/298
